includes/fields/class-field.php

```diff
@@ -565,7 +565,8 @@ public function display() {
 					'value' => $value,
 				]
 			),
-			$this->display_template
+			$this->display_template,
```

Comment:
Please also add a fallback for those with the old versions, e.g. if someone used $123 in the listing title format their website will start showing just 123 because it will stop using the display format, so we should limit this to new versions only somehow. Maybe by adding a new callback in the Upgrade component and looping over the title formats, emails etc. replacing values with formats from attributes.

Another solution is just checking the HivePress installation date (it's stored in the option). Don't forget to add the deprecation comment.